blog/_posts/2023-04-18-integrating-scalajs-with-vite.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: blog-detail
+post-type: blog
+by: Sébastien Doeraene
+title: Integrating Scala.js with Vite
+description: Using the new vite-plugin-scalajs to better integrate Scala.js with the Vite build tool.
+---
+
+One of the cornerstones of Scala.js is its ability to interoperate with JavaScript, in order to leverage its massive ecosystem.
```

Comment:
I'm going to give my usual advice here, which is to open with something that will immediately catch the reader's interest, rather than gradually leading up to it. Figure out what you'd say in a tweet, and then put that first.

Here's an example opening, not sure if it's what you want, but to illustrate the principle:

> In front end work, it's frustrating if you can't try your changes out immediately. This post is about vite-plugin-scalajs, a new plugin that can reduce your Scala.js turnaround time from 30 seconds to under 5.

Conceptually, "One of the cornerstones..." isn't part of your intro/abstract. It's the lead for your “Background/Previous Work” chapter, which comes after the intro/abstract.